se.trixon.Mapollage.yml

```diff
@@ -0,0 +1,53 @@
+id: se.trixon.Mapollage
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: start-mapollage
+separate-locales: false
+
+finish-args:
+  - --device=dri
+  - --filesystem=host
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+
+build-options:
+    env:
+        - name=mapollage
+cleanup:
+  - '/mapollage/start-mapollage'    
+  - '/mapollage/se.trixon.Mapollage.desktop'    
+  - '/mapollage/se.trixon.Mapollage.metainfo.xml'    
```

Comment:
why do you delete these. you should use them instead of putting them in the repo.

Reply:
I'll rephrase. This doesn't do what you think it does, likely do nothing.

Reply:
on the other hand you could cleanup all the unecessary windows stuff found in the archive. And riscv. and i386.

Reply:
> why do you delete these. you should use them instead of putting them in the repo.

Unless I do so they remain in $HOME/.local/share/flatpak/app/se.trixon.Mapollage/x86_64/master/active/files/mapollage/

Reply:
> on the other hand you could cleanup all the unecessary windows stuff found in the archive. And riscv. and i386.

Thanks, it's done.

Reply:
So why is the cleanup still here? If a metainfo file is provided by the package, why do you have a separate one?

Reply:
Ok, I see now... I didn't notice you were cp -r ing everything to `/app/mapollage`.